strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/domain/SystemRole.java

```diff
@@ -0,0 +1,17 @@
+package org.carlspring.strongbox.users.domain;
+
+/**
+ * Defines the restricted system roles. The purpose is to restrict changing or
+ * deleting some of the configured roles.
+ * 
+ * @author sbespalov
+ */
+public enum SystemRole
+{
+    ADMIN,
```

Comment:
I think we'll need to have a way to have descriptions for these (and possibly more) so that these could be visualized in the UI.

@steve-todorov : What are your thoughts and recommendations for this?

Reply:
Yes, definitely! I was about to add this as a comment as well. The `Roles/SystemRoles` and `Authorities` should have a `description` so that we can display this in the front-end for the admin to understand what he's assigning to a user.

Reply:
@steve-todorov @carlspring there is a role descriptions within `strongbox-authorization.yaml`, all roles should be defined there.

this `enum` needed only to know what roles assumed to be for system purpose

Reply:
I see now. Thanks for the explanation! :) 
Can we add a description for the `Authorities` as well in that case?

Reply:
@steve-todorov you mean`Privileges` enum? :smiley:

Reply:
Yeah, sorry, I meant the `Privileges` enum. I have always been calling it `Authorities`, because that's what it's called in Spring. :D